xwiki-commons-core/xwiki-commons-diff/xwiki-commons-diff-display/src/test/java/org/xwiki/diff/display/internal/DefaultUnifiedDiffDisplayerTest.java

```diff
@@ -533,4 +533,19 @@ private static String readContent(String fileName) throws IOException
         BufferedReader reader = new BufferedReader(new InputStreamReader(stream));
         return IOUtils.toString(reader);
     }
+
+    @Test
+    void displayWithConflictsOther() throws Exception
+    {
+        List<String> previous = readLines("integration3/previous.txt");
+        List<String> current = readLines("integration3/current.txt");
+        List<String> next = readLines("integration3/next.txt");
+        MergeResult<String> mergeResult = getDiffManager().merge(previous, next, current, null);
+        assertEquals(3, mergeResult.getConflicts().size());
+
+        DiffResult<String> diffResult = getDiffManager().diff(previous, current, null);
+        List<UnifiedDiffBlock<String, Object>> unifiedDiffBlocks = unifiedDiffDisplayer.display(diffResult);
+
+        unifiedDiffBlocks = unifiedDiffDisplayer.display(diffResult, mergeResult.getConflicts());
```

Comment:
Is something missing? Just wondering why this variable is affected but never used afterward.